.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+-   id: check-manifest
```

Comment:
My YAML linter complains

> hyphens: too many spaces after hyphen

(I can fix that after merging if you don't want to.)

Reply:
I know and it was weird for me, too. I think this is just how the author formats the yaml. See here the following two examples: https://pre-commit.com/#docker_image. If you want a different format, it is fine.